client/blocks/reader-combined-card/post.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { has, get } from 'lodash';
+import ReactDom from 'react-dom';
+import closest from 'component-closest';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import AutoDirection from 'components/auto-direction';
+import ReaderExcerpt from 'blocks/reader-excerpt';
+import ReaderVisitLink from 'blocks/reader-visit-link';
+import ReaderAuthorLink from 'blocks/reader-author-link';
+import { recordPermalinkClick } from 'reader/stats';
+import PostTime from 'reader/post-time';
+import ReaderFeaturedImage from 'blocks/reader-featured-image';
+import ReaderFeaturedVideo from 'blocks/reader-featured-video';
+import * as stats from 'reader/stats';
+
+class ReaderCombinedCardPost extends React.Component {
+	static propTypes = {
+		post: React.PropTypes.object.isRequired,
+		streamUrl: React.PropTypes.string,
+		onClick: React.PropTypes.func,
+	};
+
+	propagateCardClick = () => {
+		// If we have an discover pick post available, send the discover pick to the full post view
+		const postToOpen = get( this.props, 'discoverPick.post' ) || this.props.post;
+		this.props.onClick( postToOpen );
+	}
+
+	handleCardClick = ( event ) => {
+		const rootNode = ReactDom.findDOMNode( this ),
+			selection = window.getSelection && window.getSelection();
+
+		// if the click has modifier or was not primary, ignore it
+		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
+			if ( closest( event.target, '.reader-combined-card__post-title-link', true, rootNode ) ) {
+				stats.recordPermalinkClick( 'card_title_with_modifier', this.props.post );
+			}
+			return;
+		}
+
+		if ( closest( event.target, '.should-scroll', true, rootNode ) ) {
```

Comment:
I'm pretty confident `should-scroll` is no longer used... 

Good explanation of `ignore-click` and why we still require it